### PR TITLE
Remove random flip in load_image_train()

### PR DESCRIPTION
**Pull request by netw0rkf10w**
Because the results of this function are cached (with `train.cache()`), I think it doesn't make sense to do random augmentation here. If one wants to keep this then it would be necessary to split `load_image_train()` into two separate parts (e.g. `load_resize()` and `random_flip()`) and modify the mapping as:
```
train = dataset['train'].map(load_resize, num_parallel_calls=tf.data.experimental.AUTOTUNE)
train = train.cache()
train = train.map(random_flip)
train_dataset = train.map(random_flip).shuffle(BUFFER_SIZE).batch(BATCH_SIZE).repeat()
train_dataset = train_dataset.prefetch(buffer_size=tf.data.experimental.AUTOTUNE)
```

### CODE REVIEW

**Comment by yashk2810** [comment]
What's the performance hit if the `.cache` is removed?

**Comment by MarkDaoust** [comment]
@netw0rkf10w thanks for the PR. Can you split the load function as you described?

**Comment by netw0rkf10w** [comment]
@yashk2810 @MarkDaoust Thanks for your replies. I will review this PR next week as I'm working on a deadline.

**Comment by netw0rkf10w** [comment]
Another patch for the tutorial: when resizing the mask, on should use `method=tf.image.ResizeMethod.NEAREST_NEIGHBOR`. This one is more critical than the above. I'll create another PR (I'm just noting here so that I won't forget).

**Comment by lamberta** [comment]
PR stalled and out of sync. Thanks, though
